File: python_programming_basics/conditional_statements_advanced/l_08_cinema_ticket.py
day_of_week = input()
ticked_price = 0

if day_of_week == 'Monday' or day_of_week == 'Tuesday' or day_of_week == 'Friday':
    ticked_price = 12
    print(ticked_price)

elif day_of_week == 'Wednesday' or day_of_week == 'Thursday':
    ticked_price = 14
    print(ticked_price)

elif day_of_week == 'Saturday' or day_of_week == 'Sunday':
    ticked_price = 16
    print(ticked_price)

File: JavaScript Basics/03. Conditional Statements Advanced/l-08-Cinema-Ticket.js
function cinemaTicketPrice(day) {
    let price;

    switch (day) {
        case "Monday":
        case "Tuesday":
        case "Friday":
            price = 12;
            break;
        
        case "Wednesday":
        case "Thursday":
            price = 14;
            break;
        
        case "Saturday":
        case "Sunday":
            price = 16;
            break;
        
        default:
            console.log("Invalid day");
            return;
    }

    console.log(price);
}

cinemaTicketPrice("Monday");